shop/src/main/java/ro/msg/learning/shop/repository/LocationRepository.java

```diff
@@ -0,0 +1,12 @@
+package ro.msg.learning.shop.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import ro.msg.learning.shop.domain.Location;
+import java.util.UUID;
+@Repository
+public interface LocationRepository extends JpaRepository<Location, UUID> {
+
```

Comment:
this space is bigger than my living room. 
keeping code clean and organized is also important, not just for the reviewer, but also for you and your collagues. 
Whenever we respect a higher standard, we set a higher standard for everyone else. Thus the code will improve.
P.S. advice can also be applied in life in general

Reply:
Removed